toolbox/fdc3-workbench/src/fixtures/contexts.ts

```diff
@@ -6,6 +6,138 @@ import { ContextItem } from "../store/ContextStore";
 import { v4 as uuidv4 } from "uuid";
 
 export const contexts: ContextItem[] = [
+	{
+		uuid: uuidv4(),
+		id: "Chart example",
+		template: {
+			type: "fdc3.chart",
+			instruments: [
+				{
+					type: "fdc3.instrument",
+					id: {
+						ticker: "AAPL"
+					}
+				},
+				{
+					type: "fdc3.instrument",
+					id: {
+						ticker: "GOOG"
+					}
+				}
+			],
+			range: {
+				type: "fdc3.timeRange",
+				startTime: "2020-09-01T08:00:00.000Z",
+				endTime: "2020-10-31T08:00:00.000Z"
+			},
+			style: "line"
+		},
+		schemaUrl: new URL("https://fdc3.finos.org/schemas/next/context/chart.schema.json"),
```

Comment:
Should these be next or should it be a specific version to make sure they aren't changed underneath?

Reply:
Given this is existing pattern, it doesn't matter for this change but a perhaps a future discussion.

Reply:
It's worth a future discussion - as the general rules around context don't change often I (personally) don't think the tie to the same standard version and cycle as the API(s) makes sense and hence that it would be better if we can pull in the examples from the standard.

There's an issue representing that question at https://github.com/finos/FDC3/issues/940 and we've started taking steps that might enable us to move to a different system of governance, e.g. reducing the work you have to do to submit a type (down to just a single schema file that contains the schema, docs and examples - see: https://github.com/finos/FDC3/issues/1068). We could eventually build a tool that would help produce the schema from an example pasted into a form, buy using one of several libs that will propose a JSONSchema based on an example JSON.  We could then enable (and require!) the addition of docs etc. and make it really simple and quick to submit a proposal PR, automate publishing or proposed, standard and even proprietary types...